crates/runtimelib/src/messaging/mod.rs

```diff
@@ -214,11 +220,22 @@ pub struct JupyterMessage {
     pub parent_header: Option<Header>,
     pub metadata: Value,
     pub content: JupyterMessageContent,
-    #[serde(skip_serializing, skip_deserializing)]
+    #[serde(serialize_with = "serialize_base64", skip_deserializing)]
```

Comment:
This is specific to our wry window handling? Normally they come in as an array of u8 on the front end?

I'm beginning to think we'll need to setup a trait or enum to abstract the different modeling of Jupyter messages, especially as I need to break up more from runtime lib into structures that can be used by the websocket client.